README.md

```diff
@@ -0,0 +1,7 @@
+# SimpleITKUtilities
+
+![SimpleITK Utilities Testing](https://github.com/SimpleITK/SimpleITKUtilities/actions/workflows/main.yml/badge.svg)&nbsp;&nbsp;[![SimpleITK Utilities Website](https://img.shields.io/website-up-down-brightgreen-red/http/shields.io.svg)](http://simpleitk.org/SimpleITKUtilities/)
+
```

Comment:
Need to link to the Sphinx docs: http://simpleitk.org/SimpleITKUtilities/

Should probably link it SimpleITK/ITK too.